slack_sdk/http_retry/builtin_handlers.py

```diff
@@ -56,7 +56,7 @@ def _can_retry(
         response: Optional[HttpResponse] = None,
         error: Optional[Exception] = None,
     ) -> bool:
-        return response.status_code == 429
+        return response is not None and response.status_code == 429
```

Comment:
While running tests, some tests detected a potential error pattern where the response can be None when error is thrown.